Lib/idlelib/idle_test/test_debugger.py

```diff
@@ -23,7 +34,456 @@ def test_init(self):
         debugger.NamespaceViewer(self.root, 'Test')
 
 
-# Other classes are Idb, Debugger, and StackViewer.
+class MockFrameType(object):
```

Comment:
IDLE tests aren't backported to 2.7, so you don't need to include object when defining a class.